plugins/CoreAdminHome/lang/en.json

```diff
@@ -45,7 +45,7 @@
         "JSTrackingIntro5": "If you want to do more than track page views, please check out the %1$sMatomo Javascript Tracking documentation%2$s for the list of available functions. Using these functions you can track goals, custom variables, ecommerce orders, abandoned carts and more.",
         "JSTracking_CampaignKwdParam": "Campaign Keyword parameter",
         "JSTracking_CampaignNameParam": "Campaign Name parameter",
-        "JSTracking_CodeNoteBeforeClosingHead": "Make sure this code is on every page of your website. We recommend pasting it immediately before the closing %1$s tag.",
+        "JSTracking_CodeNoteBeforeClosingHead": "Click on the button to copy the code snippet and paste it immediately before the closing %1$s tag (recommended)",
```

Comment:
Would this warrant a new translation key? Sounds different enough to me, but I'm on the fence here.

Reply:
The question that needs to be answered in such a case is: "Does it have a benefit if an old translation would be used in that place". In that case I would actually say it might be fine to use the same one. Could be better to have the old text being displayed translated  instead of the new one in English.

Reply:
Nice, that's a good way how to reason about it 👍, and you're right, it's quite possible the original message translated might be better for non-English speaking users than the new one in English only.